source/_guides/05-stack-configuration-best-practices.md

```diff
@@ -18,6 +18,23 @@ what they are and for what they're good and for what not. We're sure with this y
 At the end of the article, you'll find a complete example of all the recommendations here. You can pick the things you 
 want or apply to your use cases and leave the other things out. 
 
+### Get the "original" image without scaling
+
+While you can use the `dynamic` route without any parameters to get the image in the size as it was uploaded, we recommend
+to create a stack dedicated to that purpose. Create the stack without any operations. We recommend to enable the `autoformat`
+option to have the stack deliver webp if supported by the client:
+
+```language-json
+{
+  "options": { "autoformat": true }
+}
+```
+
+We like to call this stack `original`.
+
+Note: This stack (nor the `dynamic` route) return you the exact same binary data of the image. The image size is preserved
+but rokka re-encodes the image to optimize size and quality.
```

Comment:
is there a configuration option to enable unprocessed output, if that is desired? or is that only possible for authenticated users through the api? (if we explain that somewhere else, i think i best insert a reference to that discussion rather than repeat it here)